Bumbodium.Data/Repositories/EmployeeRepo.cs

```diff
@@ -21,6 +21,10 @@ public Employee GetEmployee(string id)
         {
             return _ctx.Employee.Where(e => e.EmployeeID == id).Single();
         }
+        public Employee GetEmployeeByName(string name)
+        {
+            return _ctx.Employee.Where(e => e.FullName == name).Single();
+        }
```

Comment:
Employee names are not unique, so this method fails when theres multiple people with the same name.

Wherever you used this method, replace it with the normal GetEmployee method.